digital-islandupdate.sh

```diff
@@ -123,7 +126,7 @@ git clone https://github.com/utkdigitalinitiatives/islandora_datastream_replace
 rm -R islandora
 git clone git://github.com/Islandora/islandora
 
-#- islandora_solution_pack_collection
+#- our version of islandora_solution_pack_collection
```

Comment:
Just wondering, why do we keep our own copy of this?  I know we modify the .tpls, but those are overwritten in the theme layer, not in the actual module.  Just trying to remember.

Reply:
the default sort of listings is by title,  we replace that by a sort on PID

Reply:
yeah, just found it: https://github.com/utkdigitalinitiatives/islandora_solution_pack_collection/commit/081523e4405eb6b0a9ea6bac300adf8742a4145d

Reply:
So i'm good. Merging.